Microsoft.Toolkit.Uwp.UI/Extensions/ListViewBase/ListViewExtensions.SmoothScrollIntoView.cs

```diff
@@ -80,20 +80,7 @@ public static async Task SmoothScrollIntoViewWithIndexAsync(this ListViewBase li
             // Scrolling back to previous position
             if (isVirtualizing)
             {
-                var tcs = new TaskCompletionSource<object>();
-
-                void ViewChanged(object obj, ScrollViewerViewChangedEventArgs args) => tcs.TrySetResult(result: null);
-
-                try
-                {
-                    scrollViewer.ViewChanged += ViewChanged;
-                    scrollViewer.ChangeView(previousXOffset, previousYOffset, zoomFactor: null, disableAnimation: true);
-                    await tcs.Task;
-                }
-                finally
-                {
-                    scrollViewer.ViewChanged -= ViewChanged;
-                }
+                await scrollViewer.ChangeViewAsync(previousXOffset, previousYOffset, zoomFactor: null, disableAnimation: true).ConfigureAwait(true);
             }
 
             var listViewBaseWidth = listViewBase.ActualWidth;
```

Comment:
I wanted to see if this can be abstracted so this and `ChangeViewAsync` shared more of the same code path, but I could not figure out anything that clean. So it should be fine to have this repeating pattern.